.github/workflows/matrix.yml

```
@@ -66,34 +66,19 @@ jobs:
         native: [linux64]
 
         include:
+          - compiler: clang++-12
+            os: ubuntu-latest
+            sanitize: undefined
```

Let's get early feedback about magiclysm load errors.
```suggestion
            sanitize: undefined
            mods: --mods=magiclysm
```